### PR TITLE
Update pull_request_checklist.md

### PR DESCRIPTION
**Pull request by KrunalKharat02**
dont know much of languange you are using in project but still tried to contribute. i am participating in hacktoberfest i do need this.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-## Describe your changes
+## Describe your changes(in 200 words)
```
Comment (proffapt) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md:1]: No need for this, I prefer long descriptions

File: .github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-## Issue ticket number and link
+## Issue ticket number and link
```
Comment (proffapt) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md:3]: Only a space changed

**Review by proffapt — COMMENTED** [review]
Unnecessary changes.

**Comment by proffapt** [comment]
Do checkout other projects, and see if you can contribute to them.
I won't be able to apply it for hacktoberfest.
And have to mark it as `invalid`.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md (hunk 1 above)
Comment (KrunalKharat02) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md:1]: okk i dont have any knowlege in the domain

Comment (proffapt) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md:1]: Learn then.

**Pull request closed without merging**